src/app/modules/item/components/sub-skills/sub-skills.component.html

```diff
@@ -1,45 +1,30 @@
 <ng-container *ngIf="state$ | async as state">
-  <alg-section i18n-label label="Sub-skills" icon="fa fa-graduation-cap">
+  <alg-loading *ngIf="state.isFetching"></alg-loading>
 
-    <alg-loading *ngIf="state.isFetching"></alg-loading>
+  <alg-error
+    *ngIf="state.isError"
+    class="dark"
+    i18n-message message="Error while loading content"
+    [showRefreshButton]="true"
+    (refresh)="refresh()"
+  ></alg-error>
 
-    <alg-error
-      *ngIf="state.isError"
-      class="dark"
-      i18n-message message="Error while loading content"
-      [showRefreshButton]="true"
-      (refresh)="refresh()"
-    ></alg-error>
+  <ng-container *ngIf="state.isReady && state.data as children">
```

Comment:
so you do not use `children` and the data in practice... see next comment

Reply:
Fixed